policy-integrations/app.py

```diff
@@ -0,0 +1,194 @@
+from flask import Flask, request, jsonify, abort, make_response
```

Comment:
remove commented out code, and use constants for `ip`, `protocol`, `email`, etc...